docs/README.md

```diff
@@ -0,0 +1,227 @@
+# API Specification
```

Comment:
I think we will want the first docs page to be a little more general / about the library, maybe some quick usage samples, etc.

Let's reorg this file into `docs/specification/README.md`

Reply:
Created https://github.com/cal-itp/eligibility-api/issues/9 for this